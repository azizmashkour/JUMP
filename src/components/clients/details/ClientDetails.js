import React from 'react';
import PropTypes from 'prop-types';
import { withTranslation } from 'react-i18next';

const ClientDetails = (props) => {

  return (
    <fieldset>
        <legend>{props.t('client_details.details')}</legend>
        <p>{props.t('client_details.address')}: <strong> {props.address} </strong></p>
    </fieldset>
  );
};

ClientDetails.propTypes = {
  t: PropTypes.func.isRequired,
};

export default withTranslation()(ClientDetails);
