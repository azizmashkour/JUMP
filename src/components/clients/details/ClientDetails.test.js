import React from "react";
import { shallow } from "enzyme";
import ClientDetails from "./ClientDetails";

const details = "Details";
const address = "Address";

describe('<ClientDetails />', () => {
  it("should display ClientDetails component", () => {
    const wrapper = shallow(<ClientDetails />);
    expect(wrapper.find("fieldset")).toBeTruthy();
    expect(wrapper.find("fieldset>legend")).toBeTruthy();
    expect(wrapper.find("fieldset>p")).toBeTruthy();
    expect(wrapper.find(details)).toBeTruthy();
    expect(wrapper.find(address)).toBeTruthy();
    expect(wrapper).toMatchSnapshot();
  });
});
