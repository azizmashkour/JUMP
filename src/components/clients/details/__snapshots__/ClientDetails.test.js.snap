// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ClientDetails /> should display ClientDetails component 1`] = `ShallowWrapper {}`;
